src/shogun/evaluation/CrossValidation.cpp

```diff
@@ -183,9 +184,9 @@ float64_t CCrossValidation::evaluate_one_run(
 			    m_splitting_strategy->generate_subset_indices(i);
 
 			/* evtl. update xvalidation output class */
-			fold->set_train_indices(inverse_subset_indices);
+			fold->put("train_indices", inverse_subset_indices);
 			auto fold_machine = (CMachine*)m_machine->clone();
-			fold->set_trained_machine(fold_machine);
+			fold->put("trained_machine", fold_machine);
```

Comment:
same here just "machine" Keep in mind that these fields will (hopefully) have meaningful descriptions that would in this case say "Machine trained on training set"

Reply:
same as above :) even if we have the descriptions